src/main/java/pl/simpleascoding/tutoringplatform/service/user/UserServiceImpl.java

```diff
@@ -26,17 +26,18 @@
 @RequiredArgsConstructor
 class UserServiceImpl implements UserService {
 
-    private final UserRepository userRepository;
-    private final TokenRepository tokenRepository;
-    private final PasswordEncoder passwordEncoder;
-
-    private final JavaMailSender mailSender;
-
     private static final String REGISTRATION_MAIL_SUBJECT = "Confirm your email";
     private static final String REGISTRATION_MAIL_TEXT = "Hi %s, please visit the link below to confirm your email address and activate your account: \n%s";
     private static final String REGISTRATION_LINK = "%s/confirm-registration?tokenValue=%s";
-
     private static final String USER_NOT_FOUND_MSG = "User with \"%s\" username, has not been found";
+    private static final String CONFIRMATION_MAIL_SUBJECT = "Confirm your password change";
+    private static final String CONFIRMATION_MAIL_TEXT = "Hi %s, please visit the link below to confirm your password change: \n%s";
+    private static final String CONFIRMATION_LINK = "%s/confirm-change-password?tokenValue=%s";
+
```

Comment:
These string names imply, that they are used for every confirmation email that will be sent from our app.
This is not true, as we will require the user to confirm their email change as well.
Please rename these fields to something that will differentiate them from other (future) confirmation strings.